src/sentry/incidents/logic.py

```diff
@@ -854,6 +860,12 @@ def update_alert_rule(
             updated_fields["seasonality"] = None
         elif detection_type == AlertRuleDetectionType.DYNAMIC:
             updated_fields["comparison_delta"] = None
+            if (
+                time_window not in {15, 30, 60}
```

Comment:
lets make these windows constants
maybe something like
`window_s`, `window_m`, `window_l`
and `window_s_min`, `window_m_min`, `window_l_min`?
I dunno - maybe something else if we ever anticipate more windows than these